sys-apps/edac-utils/edac-utils-0.18-r1.ebuild

```diff
@@ -1,27 +1,41 @@
 # Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=7
 
-inherit eutils ltprune
+inherit autotools flag-o-matic
 
 DESCRIPTION="Userspace helper for Linux kernel EDAC drivers"
 HOMEPAGE="https://github.com/grondo/edac-utils"
 SRC_URI="https://github.com/grondo/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="amd64"
+KEYWORDS="~amd64"
 IUSE="debug"
 
 DEPEND="sys-fs/sysfsutils"
 RDEPEND="${DEPEND}
 	sys-apps/dmidecode"
 
 src_prepare() {
+	default
+
+	# <sys-devel/gcc-10, <sys-devel/binutils-2.34 do not support LDPT_ADD_SYMBOLS_V2
+	# https://sourceware.org/bugzilla/show_bug.cgi?id=25355
+	local gcc_vers=$(gcc-version)
+	local ld_vers=$(ld.bfd -v | awk '{ print $4 }')
```

Comment:
Do we know ld.bfd always exists?

Reply:
>Do we know ld.bfd always exists?

Cases where it won't exist: the system has a nonstandard arch or sys-devel/binutils-config is built with USE=-native-symlinks.

With regards to multiarch compat, I think the ~amd64 keyword is sufficient for the time being, given the homepage's description. Beyond that, I don't have much experience with how different architectures handle things.

`[...]these modules mainly handle detection of ECC memory errors for many
x86 and x86-64 chipsets and PCI bus parity errors.`

---

Regarding ld.bfd; would this be better? `$(binutils-config -B)##*/}`

And thanks for the rebase tip, I'll look into it!